src/plugins/intel_npu/src/plugin/npuw/partitioning/patterns/dcoff.cpp

```diff
@@ -115,25 +116,44 @@ ClosureRemap build_remap(const Function& fbody, const DCOFFParams& params_to) {
 
 void apply_remap(Subgraph& fcall, const ClosureRemap& m) {
     std::vector<ov::Tensor> new_closure;
+    std::vector<ov::npuw::weights::LazyTensor> new_lazy_closure;
     std::vector<ov::Tensor> new_scales;
     std::vector<ov::Tensor> new_zerops;
 
-    // For a new_closure vector by rearranging the old one.  Also
+    // For a new_lazy_closure vector by rearranging the old one.  Also
     // reserve a new_scales vector to have the same size, filled with
     // empty tensors by default.
     for (auto&& i : m.closure_remap) {
+        new_lazy_closure.push_back(fcall._lazy_closure[i]);
         new_closure.push_back(fcall._closure[i]);
 
         auto scale_iter = m.scale_remap.find(i);
-        new_scales.push_back(scale_iter != m.scale_remap.end() ? fcall._closure[scale_iter->second] : ov::Tensor());
-        // Check for asymmetric zero points and add them to new_zerops
         auto zerop_iter = m.zerop_remap.find(i);
-        const auto& zerop = zerop_iter != m.zerop_remap.end() ? fcall._closure[zerop_iter->second] : m.zero_points[i];
-        new_zerops.push_back(zerop);
+        // FIXME: assuming no transformations were applied to the tensor - since we are utilizing the original
+        // ov::Tensor below
+        if (scale_iter != m.scale_remap.end()) {
+            NPUW_ASSERT(!fcall._lazy_closure[scale_iter->second].has_transformations());
```

Comment:
It actuall can have transformations, it is ok. We may be asked to transpose a GQ tensor even for the DCOFF case.

Reply:
Done